src/lib/bankImage.ts

```diff
@@ -676,6 +671,94 @@ class BankImageTask {
 	}
 }
 
+const chestLootTypes = [
+	{
+		title: 'Tombs of Amascut',
+		chestImage: loadImage('./src/lib/resources/images/toaChest.png'),
+		chestImagePurple: loadImage('./src/lib/resources/images/toaChestPurple.png'),
+		width: 240,
+		height: 220,
+		purpleItems: toaPurpleItems
+	}
+] as const;
+
+interface CustomText {
+	x: number;
+	y: number;
+	text: string;
+}
+export async function drawChestLootImage(options: {
+	entries: { previousCL: Bank; user: MUser; loot: Bank; customTexts: CustomText[] }[];
+	type: typeof chestLootTypes[number]['title'];
+}) {
+	const type = chestLootTypes.find(t => t.title === options.type);
+	if (!type) throw new Error(`Invalid chest type: ${options.type}`);
+
+	const canvases: Canvas[] = [];
+
+	let anyoneGotPurple = false;
+
+	for (const { previousCL, loot, user, customTexts } of options.entries) {
+		const canvas = new Canvas(type.width, type.height);
+		const ctx = canvas.getContext('2d');
+
+		const { sprite } = bankImageGenerator.getBgAndSprite();
+
+		ctx.imageSmoothingEnabled = false;
+		ctx.fillStyle = ctx.createPattern(sprite.repeatableBg, 'repeat')!;
+		ctx.fillRect(0, 0, canvas.width, canvas.height);
+		const isPurple: boolean = loot.items().some(([item]) => type.purpleItems.includes(item.id));
+		if (isPurple) anyoneGotPurple = true;
+		const image = isPurple ? await type.chestImagePurple : await type.chestImage;
+		ctx.drawImage(image, canvas.width - image.width + 25, 44 + canvas.height / 4 - image.height / 2);
+
+		drawTitle(ctx, `${user.rawUsername} (${toKMB(loot.value())})`, canvas);
```

Comment:
Make this (Value: ${toKMB(loot.value())}) so that it's same as other bank images.